lib/fluent/supervisor.rb

```diff
@@ -424,6 +424,10 @@ def dump
   end
 
   class Supervisor
+    # For ServerEngine's `reload_config`.
+    # This is called only at the initilization of the supervisor process,
+    # since Fluentd overwrites all related SIGNAL(HUP,USR1,USR2) and have own
+    # reloading feature.
     def self.load_config(path, params = {})
```

Comment:
Perhaps we should rename or move this method.
It is hard to understand what this method is.
I will do it in another PR.